src/tests/runtests.rs

```diff
@@ -21238,7 +21238,10 @@ unsafe extern "C" fn test_nsalloc_realloc_long_prefix() {
     let mut i: c_int = 0;
-    let max_realloc_count: c_int = 12;
+
+    // REXPAT: max_realloc_count was 12. We realloc the main buffer rather than allocing, so
+    // that adds two reallocs to the parse.
+    let max_realloc_count: c_int = 13;
```

Comment:
Could raise this to 15 just to be safe.

Reply:
Don't really care, but I would like to know if we trigger any new reallocs.